examples/energy-management-app/energy-management-common/energy-reporting/src/FakeReadings.cpp

```diff
@@ -180,3 +180,60 @@ void FakeReadings::FakeReadingsTimerExpiry(System::Layer * systemLayer, void * m
 
     mn->FakeReadingsUpdate();
 }
+
+/**
+ * @brief   Sets the fake Power level to use
```

Comment:
nit: these comments seem to just waste vertical space. It is super obvious that a call of `void SetFoo(foo_unit value)` "sets the value of Foo in unit units".

Reply:
Agree - ok removing them.